### ISSUE TITLE
Can't push to Heroku (LoadError: cannot load such file -- intercom-rails/intercom.rake)

### ISSUE THREAD

**Issue by rassom** [intercom/intercom-rails#209]
## Version info
  - intercom-rails version: 0.2.36
  - Rails version: 4.2.6

## Terminal output
````
remote:        Removing intercom-rails (0.2.35)
remote: sh: 2: Syntax error: Unterminated quoted string
remote: sh: 2: Syntax error: Unterminated quoted string
remote:  !
remote:  !     Could not detect rake tasks
remote:  !     ensure you can run `$ bundle exec rake -P` against your app
remote:  !     and using the production group of your Gemfile.
remote:  !     rake aborted!
remote:  !     LoadError: cannot load such file -- intercom-rails/intercom.rake
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `load'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `block in load'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:240:in `load_dependency'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `load'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/intercom-rails-0.2.36/lib/intercom-rails/railtie.rb:15:in `block in <class:Railtie>'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:237:in `instance_exec'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:237:in `block in run_tasks_blocks'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:245:in `each'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:245:in `each_registered_block'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:237:in `run_tasks_blocks'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/application.rb:451:in `block in run_tasks_blocks'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/engine/railties.rb:13:in `each'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/engine/railties.rb:13:in `each'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/application.rb:451:in `run_tasks_blocks'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/engine.rb:453:in `load_tasks'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:194:in `public_send'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:194:in `method_missing'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/Rakefile:7:in `<top (required)>'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/rake_module.rb:28:in `load'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/rake_module.rb:28:in `load_rakefile'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/application.rb:686:in `raw_load_rakefile'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/application.rb:96:in `block in load_rakefile'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/application.rb:178:in `standard_exception_handling'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/application.rb:95:in `load_rakefile'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/application.rb:79:in `block in run'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/application.rb:178:in `standard_exception_handling'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/lib/rake/application.rb:77:in `run'
remote:  !     /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/exe/rake:27:in `<top (required)>'
remote:  !     vendor/bundle/bin/rake:16:in `load'
remote:  !     vendor/bundle/bin/rake:16:in `<main>'
remote:  !
remote: /app/tmp/buildpacks/ruby/lib/language_pack/helpers/rake_runner.rb:102:in `load_rake_tasks!': Could not detect rake tasks (LanguagePack::Helpers::RakeRunner::CannotLoadRakefileError)
remote: ensure you can run `$ bundle exec rake -P` against your app
remote: and using the production group of your Gemfile.
remote: rake aborted!
remote: LoadError: cannot load such file -- intercom-rails/intercom.rake
remote: /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `load'
remote: /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `block in load'
remote: /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:240:in `load_dependency'
remote: /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `load'
remote: /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/intercom-rails-0.2.36/lib/intercom-rails/railtie.rb:15:in `block in <class:Railtie>'
remote: /tmp/build_fc9b64877c3a73a6b28b00fa3937149d/vendor/bundle/ruby/2.3.0/gems/railties-4.2.6/lib/rails/railtie.rb:237:in `instance_exec'
````
Any idea why this happens?

**Comment by rassom** [comment]
Anyone?

**Comment by Skaelv** [comment]
Hey. This has been fixed in 0.3.x you should upgrade the gem.

**Comment by rassom** [comment]
Thank you - it works now 🎉
